owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/logging/LogBuilder.kt

```diff
@@ -38,5 +39,6 @@ fun MediaType?.isLoggable(): Boolean =
         (mediaType.type == "text" ||
                 mediaTypeString.contains(CONTENT_TYPE_XML) ||
                 mediaTypeString.contains(CONTENT_TYPE_JSON) ||
-                mediaTypeString.contains(CONTENT_TYPE_WWW_FORM))
+                mediaTypeString.contains(CONTENT_TYPE_WWW_FORM) ||
+                mediaTypeString.contains(CONTENT_TYPE_JRD_JSON))
```

Comment:
Update the documentation in the header of the class as well

Reply:
Done